index.d.ts

```diff
@@ -17,6 +17,6 @@ isAbsoluteUrl('foo/bar');
 //=> false
 ```
 */
-declare function isAbsoluteUrl(url: string): boolean;
+declare function isAbsoluteUrl(url: string, options: object): boolean;
```

Comment:
This should be strongly-typed. See: https://github.com/sindresorhus/typescript-definition-style-guide/tree/57a925bb5b3eb46a4666eab31e1c883a26b7bfb5#documentation Should also have docs.